build.gradle.kts

```diff
@@ -116,6 +116,7 @@ dependencies {
     testRuntimeOnly("org.springframework:spring-test:4.+")
     testRuntimeOnly("ch.qos.logback:logback-classic:1.0.13")
     testRuntimeOnly("org.mockito:mockito-all:$mockito1Version")
+    testRuntimeOnly("org.mockito:mockito-core:latest.release")
```

Comment:
`testRuntimeOnly("org.mockito:mockito-core:latest.release")` is going to pull in Mockito 3. Are the symbols this operates on not present in mockito 1? _Some_ symbols overlap between Mockito 1 and 3, I wonder if this could cause any problems.

Reply:
I'll check on this tomorrow. The tests fail without it due to missing class info.